src/WSRequestHandler_General.cpp

```diff
@@ -171,6 +171,7 @@ RpcResponse WSRequestHandler::Authenticate(const RpcRequest& request) {
  * @name SetHeartbeat
  * @category general
  * @since 4.3.0
+ * @deprecated Since unreleased. Please poll the appropriate data using requests. Will be removed in v5.0.0.
```

Comment:
I believe it's fine to already put 4.9 here.

```suggestion
 * @deprecated Since 4.9.0. Please poll the appropriate data using requests. Will be removed in v5.0.0.
```